core/src/main/resources/reference.conf

```diff
@@ -28,6 +28,7 @@ health-monitor-dispatcher {
 }
 
 gcs {
+  defaultLocation = "us-central1"
```

Comment:
This is just the reference conf file-- doesn't the actual rawls conf template in firecloud-develop need to be updated also?

Reply:
(there's also a reference.conf in a test directory… would it make sense to define the defaultLocation there as well?)

Reply:
There are quite a few config values that aren't environment or run-context specific in the base-configs for Rawls and other services. I like that this is closer to the code, but it can also be tricky to find config values when they're defined in multiple places

Reply:
I was confused by the name "reference.conf". I thought this is just an example conf, and what firecloud-develop produces is what we actually  use.